Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/CqlTranslatorOptions.java

```diff
@@ -174,30 +186,11 @@ public void setOptions(CqlTranslator.Options... options) {
      * @param options
      * @return
      */
-    public CqlTranslatorOptions withOptions(CqlTranslator.Options... options) {
+    public CqlTranslatorOptions withOptions(Options... options) {
         setOptions(options);
         return this;
     }
 
-    /**
-     * Returns instance of CqlTranslatorOptions formats
-     *
-     * @return
-     */
-    public List<CqlTranslator.Format> getFormats() {
```

Comment:
This is used as part of the publication process to determine the ELM that FHIR Libraries should be generated with. Not sure we can remove it completely. Several IGs are already using this.

It also raises the question of "CompilerOptions" vs "TranslatorOptions". Not sure we necessarily have to do anything about that, just raising it as a consideration.

Reply:
Are people using this? I deleted it because the Format parameter doesn't do anything inside the Compiler or translator. The format is generally passed as another parameter in the call, not as an option of the Translator or Compiler. But I can bring it back.

Reply:
Yes, this is being used in the IG publisher. @brynrhodes would have more details. I'm open to alternatives for setting this.

Reply:
Yes, the translator itself doesn't do anything with it, but it's an important option that the environments use to control serialization outputs. It's definitely a "translator" option, not a "compiler" option, but I'm not sure how much mileage we'd actually get out of separating those options at this point.

Reply:
Cool, no worries. I will revert the change.